examples/gp/linear_verif/gpmsa_input.txt

```diff
@@ -0,0 +1,69 @@
+###############################################
+# UQ Environment
+###############################################
+#env_help                = anything
+env_numSubEnvironments   = 1
+env_subDisplayFileName   = outputData/display
+env_subDisplayAllowAll   = 0
+env_subDisplayAllowedSet = 0
+env_displayVerbosity     = 2
+env_syncVerbosity        = 0
+env_seed                 = -1
+
+###############################################
+# Statistical inverse problem (ip)
+###############################################
+#ip_help                 = anything
+ip_computeSolution      = 1
+ip_dataOutputFileName   = outputData/sipOutput
+ip_dataOutputAllowedSet = 0
+
+###############################################
+# 'ip_': information for Metropolis-Hastings algorithm
+###############################################
+#ip_mh_help                 = anything
+ip_mh_dataOutputFileName   = outputData/sipOutput
+ip_mh_dataOutputAllowedSet = 0 1
```

Comment:
Vectors need single quotes around them (`'0 1'`) because`dev` uses `GetPot` to parse input files, and not `boost`.  You can still use `boost` if you want, but the default is `GetPot`.